Babylon/utils/environment.py

```diff
@@ -179,3 +179,20 @@ def extract_value_content(value: Any) -> Optional[tuple[str, Optional[str], str]
         _type, _file, _query = match_content.groups()
 
         return _type, _file, _query
+
+    def fill_template(self, template_file: str, data: dict[str, Any] = {}) -> str:
```

Comment:
What is the use of this function ? it is not called anywhere in your changes

Reply:
It was used in export environment but it's not needed anymore. I will need it in another PR but I will remove it for now.